.github/workflows/file_read_write_round_trip_verification/download_file.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright (c) 2023. Lance Developers
```

Comment:
Maybe name this `download_files` so it is clear it downloads more than one?

Reply:
fixed, deleted the file, switched to use `aws CLI`